roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install nginx
+  apt:
+    pkg: nginx
+    state: latest
+    force: yes
+
+- name: Disable default site
```

Comment:
I added in disabling the default site, so the IP isn't really directly accessible